package.json

```diff
@@ -121,8 +123,10 @@
     "prettier": "^1.16.4",
     "react": "^16.5.0",
     "react-dom": "^16.8.6",
+    "react-native": "0.61.5",
```

Comment:
is there a reason why we're a couple versions behind on RN?

Reply:
@DanC5 Yeah, I can't seem to remember exactly but it was breaking for 0.62 and 0.63. But give it a shot!!